source/documentation/common/vgpu/proc_nvidia_gpu_passthrough.adoc

```diff
@@ -107,7 +107,7 @@ When using a UEFI-based host, the target file should be [filename]`/etc/grub2-ef
 ====
 
 . Reboot the host machine.
-. Confirm that IOMMU is enabled, the host device is added to the list of pci-stub.ids, and Nouveau is blacklisted:
+. Confirm that IOMMU is enabled, the host device is added to the list of pci-stub.ids, and Nouveau is disabled:
```

Comment:
Remove second comma.
Here we have to use 'denylist'. So I guess you should change this to "Nouveau is on the denylist". 
Update callout for "rdblacklist=nouveau" in the example.